apps/chat/src/utils/app/common.ts

```diff
@@ -91,3 +91,10 @@ export const prepareEntityName = (name: string) => {
 
   return clearName;
 };
+
+export const skipEmptyAndJoin = (
```

Comment:
Looks like adding such small utility will just give more problems with consistency - let's remove it